flowcraft/generator/header_skeleton.py

```diff
@@ -27,11 +27,14 @@ def infoMap = [:]
     }
 }
 if (params.containsKey("accessions")){
-    BufferedReader reader = new BufferedReader(new FileReader(params.accessions));
-    int lines = 0;
-    while (reader.readLine() != null) lines++;
-    reader.close();
-    infoMap.put("accessions", lines)
+    // checks if params.accessions is different from null
+    if (params.accessions) {
```

Comment:
check if `params.accession` is defined rather than `null`. If it fails nextflow will properly raise an error message pointing to this issue.